youtube_dl/extractor/vvvvid.py

```diff
@@ -45,16 +47,23 @@ def _real_initialize(self):
             'https://www.vvvvid.it/user/login',
             None, headers=self.geo_verification_headers())['data']['conn_id']
 
-    def _real_extract(self, url):
-        show_id, season_id, video_id = re.match(self._VALID_URL, url).groups()
+    def _download_info(self, path, video_id):
         response = self._download_json(
-            'https://www.vvvvid.it/vvvvid/ondemand/%s/season/%s' % (show_id, season_id),
+            urljoin('https://www.vvvvid.it/vvvvid/ondemand/', path),
```

Comment:
as the base URL and the paths are known beforehand, there is no need to use `urljoin`.